src/test/projects/processors/pom.xml

```diff
@@ -12,5 +12,18 @@
 		<version>8.1.0</version>
 	</dependency>
 	</dependencies>
+	<build>
+	  <plugins>
+	    <plugin>
+		  <groupId>org.apache.maven.plugins</groupId>
+		  <artifactId>maven-compiler-plugin</artifactId>
+		  <version>3.11.0</version>
+		  <configuration>
+			<source>1.8</source>
+			<target>1.8</target>
```

Comment:
Target 8 seems wrong because spoon needs java 11.

Reply:
Addressed with https://github.com/SpoonLabs/spoon-maven-plugin/pull/124 and 74149ead82c49e14af742dc42375e0125fa68133